core/synch.h

```diff
@@ -260,12 +260,18 @@ translate_from_synchall_to_dispatch(thread_record_t *tr,
 void
 send_all_other_threads_native(void);
 
+void
+detach_cleanup_helper(thread_record_t *thread _IF_WINDOWS(bool detach_stacked_callbacks));
+
+void
+detach_set_mcontext_helper(thread_record_t *thread);
```

Comment:
Both of these are internal helpers local to synch.c so they should not be listed here and should just be static inside the .c file.

Reply:
done.